zcutil/build.sh

```diff
@@ -69,13 +69,13 @@ set -x
 eval "$MAKE" --version
 as --version
 
-ENABLE_DEBUG_REGEX='^(.*\s)?--enable-debug(\s.*)?$'
-if [[ "$CONFIGURE_FLAGS" =~ $ENABLE_DEBUG_REGEX ]]
-then
+case "$CONFIGURE_FLAGS" in
+(*"--enable-debug"*)
     DEBUG=1
-else
+;;
+(*)
     DEBUG=
-fi
+;;esac
```

Comment:
I confirmed locally (Ubuntu 20.04, for which `/bin/sh -> dash`) that this sets `DEBUG` correctly.